frontend/src/API/PostAPIHelper.ts

```diff
@@ -64,7 +65,8 @@ export default class PostAPIHelper {
             post: post,
             comments: comments,
             site: siteInfo,
-            lastCommentId: lastCommentId
+            lastCommentId: lastCommentId,
+            anonymousUser: response.anonymousUser && this.appState.cache.setUser(response.anonymousUser)
```

Comment:
I'm probably missing something but I cannot seem to find where this cache is used, `APICache` has no method for that and there's no direct access in the code

Reply:
You're right. We can probably drop it altogether.